tests/acceptance/SubscribeToMultipleListsCest.php

```diff
@@ -12,6 +12,13 @@
 require_once __DIR__ . '/../DataFactories/Segment.php';
 
 class SubscribeToMultipleListsCest {
+
+  function _before(\AcceptanceTester $I) {
+    // Clean table with subscriber IPs, so another test
+    // running in parallel won't block this subscription
```

Comment:
I guess the comment is wrong. A test running in parallel cannot affect this test in any way. I think we said it's due to some previous test if I'm not mistaken.

Reply:
Correct, I changed it